docs/specs/markdown.yml

```diff
@@ -608,3 +608,13 @@ outputs:
                           <a href=\"a#\" data-linktype=\"relative-path\">a1</a>
                      </p>"
     }
+---
+# Liquid template file missing
+inputs: 
+  docfx.yml: |
+    output:
+      json: false
+  docs/a.md: Hello `docfx`! 
+outputs: 
+  .errors.log: |
+    ["error","liquid-template-missing","The liquid template '.html.liquid' is missing","docs/a.md"]
```

Comment:
We currently treat template as code, failures in template (like javascript error) causes an unhandled exception and crashes docfx. I'm not sure if it is a good idea to introduce this error code. For instance, if a `.mta.js` is missing, we ignore the script and return the input model instead. Similarly, we can have a default implementation of `.html.liquid` that shows JSON input

Reply:
sounds reasonable